gtsam/linear/NoiseModel.h

```diff
@@ -779,6 +787,14 @@ namespace gtsam {
           double absError = std::abs(error);
           return (absError < k_) ? (1.0) : (k_ / absError);
         }
+        double residual(double error) const {
+          double absError = std::abs(error);
+          if (absError <= k_) {  // |x| <= k
```

Comment:
Not: above it is < not <=

Reply:
Actually, as per the definition of Huber loss, it should be `<=` for both.
https://en.wikipedia.org/wiki/Huber_loss#Definition